vendor.sum

```diff
@@ -171,9 +173,8 @@ github.com/matttproud/golang_protobuf_extensions v1.0.4/go.mod h1:BSXmuO+STAnVfr
 github.com/miekg/pkcs11 v1.0.3/go.mod h1:XsNlhZGX73bx86s2hdc/FuaLm2CPZJemRLMA+WTFxgs=
 github.com/miekg/pkcs11 v1.1.1 h1:Ugu9pdy6vAYku5DEpVWVFPYnzV+bxB+iRdbuFSu7TvU=
 github.com/miekg/pkcs11 v1.1.1/go.mod h1:XsNlhZGX73bx86s2hdc/FuaLm2CPZJemRLMA+WTFxgs=
+github.com/mitchellh/mapstructure v1.0.0 h1:vVpGvMXJPqSDh2VYHF7gsfQj8Ncx+Xw5Y1KHeTRY+7I=
```

Comment:
🤔 why is this 1.0.0 version still here?

Reply:
Looks like it's Notary that still has it somewhere listed;

```bash
go mod graph | grep ' github.com/mitchellh/mapstructure'
github.com/theupdateframework/notary@v0.7.1-0.20210315103452-bf96a202a09a github.com/mitchellh/mapstructure@v1.0.0
```